mlflow/spark.py

```diff
@@ -513,17 +527,43 @@ def save_model(
     )
 
 
-def _load_model(model_uri, dfs_tmpdir=None):
+def _load_model_databricks(model_uri, dfs_tmpdir):
     from pyspark.ml.pipeline import PipelineModel
 
-    if dfs_tmpdir is None:
-        dfs_tmpdir = DFS_TMP
-    tmp_path = _tmp_path(dfs_tmpdir)
+    # Download model saved to remote URI to local filesystem
+    local_model_path = _download_artifact_from_uri(model_uri)
     # Spark ML expects the model to be stored on DFS
     # Copy the model to a temp DFS location first. We cannot delete this file, as
     # Spark may read from it at any point.
-    model_path = _HadoopFileSystem.maybe_copy_from_uri(model_uri, tmp_path)
-    return PipelineModel.load(model_path)
+    fuse_dfs_tmpdir = dbfs_hdfs_uri_to_fuse_path(dfs_tmpdir)
+    os.mkdir(fuse_dfs_tmpdir)
+    # Workaround for inability to use shutil.copytree with DBFS FUSE due to permission-denied
+    # errors on passthrough-enabled clusters when attempting to copy permission bits for directories
```

Comment:
Can we factor out this copytree functionality into a more generic helper function?